src/app/components/score/score.component.ts

```diff
@@ -10,9 +17,12 @@ export class ScoreComponent {
 
   @Input()
   score: MatchScore;
-
-  hover() {
-
+  isOpened: boolean;
+  subscription: Subscription;
+  constructor(private scoresApiService: ScoresApiService) {
+    this.subscription = this.scoresApiService.getData().subscribe(isOpened => this.isOpened = isOpened);
```

Comment:
this can lead to potential memory leak, please use ngOnInit lifecycle hook for asynchronous operations.